src/zenml/integrations/pytorch_lightning/__init__.py

```diff
@@ -21,7 +21,7 @@ class PytorchLightningIntegration(Integration):
     """Definition of PyTorch Lightning integration for ZenML."""
 
     NAME = PYTORCH_L
-    REQUIREMENTS = ["pytorch_lightning<2.0.0"]
+    REQUIREMENTS = ["pytorch_lightning"]
```

Comment:
Would this change also work in `pytorch_lightning < 2.0.0`?`

Reply:
~~Yes, it should work, cause we now are defaulting the `pytorch-lightning` materializer to the way the original native `pytorch` materializer work.~~
Well we can actually just delete this whole materializer, we already have `Pytorch` one